### DISCUSSION TITLE
📑 Links to guides for React that I found Helpful! 📑

### DISCUSSION THREAD

**Discussion by handbd** [UpKeepUC/UpKeep#49] [General]
# React Materials 

### Guides and Documentation 📓 
- [React Docs](https://reactjs.org/docs/getting-started.html) - Official Guide for all things react  
- [React Bootstrap](https://react-bootstrap.github.io/) - Documentation and examples of React Bootstrap Libraries
- [Create react App](https://create-react-app.dev/) - Documentation and suggestions when starting a new react app (similar to our project but still different)

### Linked-In Learning Tutorials 🧑‍🏫 
- [Web design lite React Tutorial](https://www.linkedin.com/learning/react-for-web-designers-2/set-up-first-simple-component-2?autoSkip=true&autoplay=true&resume=false&u=2133849)
- [Full Stack React Tutorial](https://www.linkedin.com/learning/react-creating-and-hosting-a-full-stack-site-15153869/making-your-articles-list-modular?autoplay=true&resume=false&u=2133849) - Perspective on linking pages and data only using JavaScript (This is for a JavaScript Standalone App but a lot of the key concepts translate)
- [React Essential Training](https://www.linkedin.com/learning/react-js-essential-training-14836121/generating-a-project-with-create-react-app?autoplay=true&u=2133849)

### YouTube Examples 📺 
- [Building an Inventory system](https://www.youtube.com/watch?v=0UNNThVWXr0)
- [TODO list in React](https://www.youtube.com/watch?v=N8kYlimhuLw&list=LL&index=4&t=581s)

### Other interesting Sites and Articles 🗒️ 
- [React CRUD pages](https://www.red-gate.com/simple-talk/development/dotnet-development/creating-asp-net-apps-with-react/)

**Comment by thiswallz** [comment]
add a good channel too, like: https://www.youtube.com/c/TheoBrowne1017 where you can find useful and trending discussions
